docs/roadmap.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 4
+---
+
+# Roadmap
+
+Agama is a project under constant development and priorities are re-evaluated every few weeks. This
```

Comment:
It's actually every few days :D But weeks sound like we know what we are doing ;)